images/bazelbuild/Dockerfile

```diff
@@ -14,7 +14,12 @@
 
 # Includes git, gcloud, and bazel.
 FROM ubuntu:16.04
-LABEL maintainer="spxtr@google.com"
+# TODO(bentheelder): consider replacing all maintainers with a test-infra-maintainers alias of some sort
+LABEL maintainer="bentheelder@google.com"
```

Comment:
this isn't strictly necessary, but I noticed that this was still out of date. when we do a larger image cleanup in Q1 we should consider using an alias for test-infra-maintainers or something, so I added a TODO, xref: https://github.com/kubernetes/test-infra/issues/5431